src/main/java/edu/oregonstate/cs361/battleship/BattleshipModel.java

```diff
@@ -142,6 +142,20 @@ public void shootAtPlayer() {
             //end of easy mode
     }
 
+    public void shootAtPlayer_hard() {
+        //this is hard mode, shoots at random locations
+        int max = 10;
+        int min = 1;
+        Random random = new Random();
+        int randRow = random.nextInt(max - min + 1) + min;
```

Comment:
This is good. Although you could have done it differently. You can use for loop to do the same thing.